ocaml/xcp-rrdd/bin/rrdd/rrdd_server.ml

```diff
@@ -294,29 +296,27 @@ let remove_rrd (uuid : string) : unit =
    is assumed to be valid, since it is set by monitor_master. *)
 let migrate_rrd (session_id : string option) (remote_address : string)
     (vm_uuid : string) (host_uuid : string) : unit =
-  try
-    let rrdi =
-      with_lock mutex (fun () ->
-          let rrdi = Hashtbl.find vm_rrds vm_uuid in
+  with_lock mutex (fun () ->
+      match Hashtbl.find_opt vm_rrds vm_uuid with
+      | Some x ->
           debug "Sending RRD for VM uuid=%s to remote host %s for migrate"
             vm_uuid host_uuid ;
           Hashtbl.remove vm_rrds vm_uuid ;
-          rrdi
-      )
-    in
-    let transport =
-      Xmlrpc_client.(
-        SSL (SSL.make ~verify_cert:None (), remote_address, !https_port)
-      )
-    in
-    send_rrd ?session_id ~transport ~to_archive:false ~uuid:vm_uuid
-      ~rrd:rrdi.rrd ()
-  with
-  | Not_found ->
-      debug "VM %s RRDs not found on migrate! Continuing anyway..." vm_uuid ;
```

Comment:
This has been lost. Instead of an Option.iter, a `match value with ... | None ->` should be used, and a change from debug to info loglevel as well.

Reply:
Is the logline not just above, L307? Can change to info anyhow

Reply:
Yeah, missed it up ahead